charm/src/iframe/static.ts

```diff
@@ -581,13 +615,14 @@ function CounterComponent() {
 
 \`\`\`jsx
 async function fetchLLMResponse() {
-  const promptPayload = { messages: ['Hi', 'How can I help you today?', 'tell me a joke']};
-  try {
-    const result = await llm(promptPayload);
-    console.log('LLM responded:', result);
-  } catch (error) {
-    console.error('LLM error:', error);
-  }
+  // place user-level requirements in system prompt
+  const promptPayload = {
+    system: 'Translate all the messages to emojis, reply in JSON.',
```

Comment:
Explicitly explain and indicate example of how to use `system` key to mitigate against basic prompt injection issues.